pipelines/optimus/README.md

```diff
@@ -29,7 +29,7 @@
 
 
 # Optimus Pipeline Overview
-![Diagram](https://github.com/HumanCellAtlas/skylab/blob/master/pipelines/optimus/Optimus_diagram.png)
+![Diagram](Optimus_diagram2.png)
```

Comment:
Instead of deleting the original and adding a new file, it would be better to replace the original with the new copy. We don't want to create a new file every time we have a new version of the diagram.

Reply:
I updated the diagram so that it has the original name again.